entity-framework/core/managing-schemas/migrations/providers.md

```diff
@@ -74,6 +81,5 @@ if (migrationBuilder.ActiveProvider == "Microsoft.EntityFrameworkCore.SqlServer"
 }
 ```
 
-
   [1]: ../../miscellaneous/cli/index.md
```

Comment:
MarkdownLinter says no :)

Just kidding, I don't really have strong feelings about most of these, but it seems better to just follow along, otherwise we start to get arbitrary spacing all around the place?

Reply:
I'm second guessing using this style of links at all in our docs. They're optimized for reading the raw markdown, but can become tedious when editing it.

Reply:
I fully agree, we should just inline links...